requirements.txt

```diff
@@ -1,23 +1,26 @@
+# Requirements for notebooks / Binderhub
 numpy==2.2.5
 scipy==1.15.2
 matplotlib==3.10.1
 pandas==2.2.3
-patsy==1.0.1
-pyarrow==20.0.0
 scikit-learn==1.6.1
 scikit-image==0.25.2
 sympy==1.14.0
 statsmodels==0.14.4
 seaborn==0.13.2
 pytest>=8.3
```

Comment:
Is pytest really a requirement? I think we should factor out the test/build requirements from the binderhub runtime ones.